ksql-common/src/test/java/io/confluent/ksql/json/JsonMapperTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.json;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+public class JsonMapperTest {
```

Comment:
is it possible to use some reflection scanning tool to make sure that `ObjectMapper` is _only_ instantiated in `JsonMapper` so that no future usages of it come up?

Reply:
I agree this would be useful, but a quick Google search for tools that can do this came up empty. Did you have something in mind? Perhaps I'm searching with the wrong keywords.

Reply:
For the wholehearted and the brave: https://spotbugs.readthedocs.io/en/stable/implement-plugin.html (I would not fault you if you have more important things to do)